### PR TITLE
fix for issue 8457: RemoteWebDriver: Multiple files uploading method looks for files on local machine instead Selenium container

### PR DESCRIPTION
**Pull request by DvoryadkinPavel**
fix for https://github.com/SeleniumHQ/selenium/issues/8457

**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: dotnet/src/webdriver/Remote/RemoteWebElement.cs [hunk 1]
```diff
@@ -278,12 +279,15 @@ public virtual void SendKeys(string text)
...
+            foreach (var fileName in fileNames)
             {
-                text = this.UploadFile(text);
+               if (this.driver.FileDetector.IsFile(fileName))
+               {
+                 fileName = this.UploadFile(fileName));
+               }                    
             }
-
+            text = String.Join("\n",fileNames);
```
Comment (AutomatedTester) [inline dotnet/src/webdriver/Remote/RemoteWebElement.cs]: This line seems unneeded

**Review by AutomatedTester — CHANGES_REQUESTED** [review]
Could you please improved the commit message as it doesn't describe what is happening in the change. Imagine if you look at the git history and only see `Update filename.cs`

File: dotnet/src/webdriver/Remote/RemoteWebElement.cs (hunk 1 above)
Comment (DvoryadkinPavel) [reply dotnet/src/webdriver/Remote/RemoteWebElement.cs]: Agreed...line deleted

Comment (DvoryadkinPavel) [reply dotnet/src/webdriver/Remote/RemoteWebElement.cs]: PR title changed: took from issue

**Comment by DvoryadkinPavel** [comment]
> Could you please improved the commit message as it doesn't describe what is happening in the change. Imagine if you look at the git history and only see `Update filename.cs`

Done

**Review by AutomatedTester — APPROVED** [review]

**Comment by AutomatedTester** [comment]
Thanks for your PR. In future please update the commit message, as asked, and not just the PR title.

**Pull request merged**